css_tasks/task4/main.css

```diff
@@ -0,0 +1,128 @@
+/****************************
+*                           *
+*      GENERAL STYLES       *
+*                           *
+****************************/
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+:root {
+  --oddRowColor: rgb(238, 234, 234);
+  --hoverColor: rgb(255, 255, 170);
+  --tableRowHeaderColor: rgb(107, 179, 228);
+  --tableBorderColor: #ccc;
+  --tableColumnHeaderColor: aquamarine;
+}
+body {
+  width: 100%;
+}
+
+/****************************
+*                           *
+*      HEADER & INPUTS      *
+*                           *
+****************************/
+
+.schedule__header {
+  margin: 2rem;
+  text-align: center;
+}
+.schedule__input--leisure {
+  margin-left: 10rem;
+}
+.schedule__input--leisure,
+.schedule__input--sport,
+.schedule__input--knowledge {
+  cursor: pointer;
+}
+.input--leisure,
+.input--sport,
+.input--knowledge {
+  padding: 0 0.5rem;
+}
+
+/****************************
+*                           *
+*          TABLE            *
+*                           *
+****************************/
+table {
+  margin: 2rem auto 0;
+  border: 1px solid var(--tableBorderColor);
+  border-collapse: collapse;
+  overflow: hidden;
+}
+tr,
+th,
+td {
+  padding: 0.8rem 1.5rem;
+  border: 1px solid var(--tableBorderColor);
+  white-space: nowrap;
+}
+td,
+th {
+  position: relative;
+}
+thead tr th {
+  background-color: var(--tableColumnHeaderColor);
+}
+thead tr th:first-child {
+  background: none;
+}
+tbody tr:nth-child(odd) td::before {
+  background-color: var(--oddRowColor);
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -20;
+}
+tbody tr th {
+  background-color: var(--tableRowHeaderColor);
+}
+tbody tr:hover {
+  background: #ffa;
+}
+tbody tr td:hover::after {
```

Comment:
generally it is better to separate with new line every style block, it is a bit easier to read
```
tbody tr th {
  background-color: var(--tableRowHeaderColor);
}

tbody tr:hover {
  background: #ffa;
}
```